www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -793,6 +793,29 @@
         <default>0</default>
         <Required>Y</Required>
       </https_only>
+      <tls_protocols type="OptionField">
+        <multiple>Y</multiple>
+        <Sorted>Y</Sorted>
+        <OptionValues>
+          <TLSv1_2 value="TLSv1.2">TLSv1.2</TLSv1_2>
+          <TLSv1_3 value="TLSv1.3">TLSv1.3</TLSv1_3>
+        </OptionValues>
+        <Required>N</Required>
+        <default>TLSv1.2,TLSv1.3</default>
```

Comment:
setting a default that is not required seems strange, especially if this value is never empty in the GUI selection?

Reply:
empty and `TLSv1.2,TLSv1.3` give the same result. but I wanted the user to see `TLSv1.2,TLSv1.3` when adding a new server (for clarity)

Reply:
yes, which is good but not making this a migration default forces you to handle this in the template separately as well: potential error now or in the future since the two are not in sync. if it were required the edge case can be removed from template

Reply:
got it, thanks. will make this field Required and update template

Reply:
@fichtner sorry, i'm in doubt: made the field required, but I'm not sure whether it is worth changing the template or leaving it as it is now, so that under no conditions (errors?) protocols below the TLS1.2 are not assigned by nginx itself (nginx default is `TLSv1 TLSv1.1 TLSv1.2;`)?

Reply:
Template should have no fallback. Default comes from model migration. I’m really just concerned about the duplication in the template if the default changes again. So it’s better to write a template that does not depend on magic value on unset.

Reply:
got it. fixed. I hope I understood you correctly)